dandi/dandiapi.py

```diff
@@ -1223,6 +1228,30 @@ def iter_upload_raw_asset(
             self, metadata=asset_metadata, jobs=jobs, replacing=replace_asset
         )
 
+    def has_data_standard(self, data_standard: str) -> bool:
+        """
+        Returns True if the Dandiset contains one or more files of the indicated
+        standard. Otherwise, returns False. This is determined by checking for
+        the RRID of the standard  in the "dataStandard" field of the assetsSummary of
+        the dandiset.
+
+        :param data_standard: can be "NWB", "BIDS", or an RRID of a standard.
+        :type data_standard: str
+        """
+        if data_standard in DATA_STANDARD_MAP:
+            rrid = DATA_STANDARD_MAP[data_standard]
+        elif data_standard.startswith("RRID:"):
+            rrid = data_standard
+        else:
+            raise ValueError(
+                "'data_standard' must be an RRID (of form 'RRID:XXX_NNNNNNN`) or one "
+                f"of the following values: {', '.join(DATA_STANDARD_MAP.keys())}"
+            )
+        assets_summary = self.get_raw_metadata()["assetsSummary"]
```

Comment:
```suggestion
        assets_summary = self.get_raw_metadata().get("assetsSummary")
        if assets_summary is None:
            warnings.warn(f"The raw metadata of RemoteDandiset {self.identifier} does not contain 'assetsSummary'.")
```

Reply:
I guess we would need more like
```suggestion
        assets_summary = self.get_raw_metadata().get("assetsSummary")
        if assets_summary is None:
            warnings.warn(
                f"The raw metadata of RemoteDandiset {self.identifier} does not contain 'assetsSummary'. "
                f"Assuming that it does not contain {data_standard}.")
            return False
```
or alike or otherwise kaboom would happen on the following line.